src/pybind/mgr/dashboard/osd_perf.html

```diff
@@ -121,7 +121,7 @@
 
 <section class="content">
 
-    <table>
+    <!-- <table>
```

Comment:
@Saumay-Agrawal why you are commenting out these two heapmap and adding two new ones?